config/schema.go

```diff
@@ -0,0 +1,836 @@
+package config
+
+type Config struct {
```

Comment:
I think I will rename this to Values, however the package name may or may not make sense.